src/app/app.html

```diff
@@ -9,6 +9,10 @@
     <ion-content>
       <ion-list role="navigation" id="navigation-menu" aria-label="Peeveetrack Menu">
         <button role="link" menuClose ion-item *ngFor="let p of pages" (click)="openPage(p)">
+        <ion-icon name="ios-heart-outline" *ngIf="p.title == 'Favorites'"></ion-icon>
```

Comment:
I'm not sure precisely what I mean, but it just seems like a lot of repetition to only vary one thing - particularly when it's based on the page title, which must be set... somewhere?